apps/srp.c

```diff
@@ -553,6 +553,8 @@ int srp_main(int argc, char **argv)
             user = *(argv++);
         else {
```

Comment:
I am not familiar with the OpenSSL coding style, so I did a change to just fix the issue. If you want a commit to add the braces to the if I can do that.